### ISSUE TITLE
app/code/local/Mage/Core/Model/Session.php missing in tag release-0.6.3 but in modman file

### ISSUE THREAD

**Issue by dolito** [nexcess/magento-turpentine#862]
app/code/local/Mage/Core/Model/Session.php

is missing in tag "release-0.6.3"
https://github.com/nexcess/magento-turpentine/tree/release-0.6.3/app/code

but included in modman file
https://github.com/nexcess/magento-turpentine/blob/release-0.6.3/modman

shouldn't be like that, should it?

**Comment by razbakov** [comment]
i didn't find any commit, which composer able to install

**Comment by razbakov** [comment]
ah! fixed!

actually  it is packages.firegento.com bug: https://github.com/magento-hackathon/composer-repository/issues/290

as temporary fix add this to your composer.json:

```
"repositories": [
        {
            "type": "git",
            "url": "https://github.com/nexcess/magento-turpentine"
        }
]
```

**Comment by dolito** [comment]
So, was not sure if I saww it correctly. Just checkout version 0.6.2 with composer -> everything ok
version 0.6.3 won't work, will hang on modman trying to copy a file that does not exist anymore

I checked the commits now, on 23.06.2015 14:59:05:

SHA-1: a0a8267c8c8b2f7771a197caa2739f08c3471988

* Merge pull request #499 from eth8505/session-rewrite

Use rewrite instead of local Session model for form key handling

@miguelbalparda forgot to change the modman file if I see it right?

**Comment by aricwatson** [comment]
Unfortunately, we fixed the modman issue after tagging for 0.6.3. If you pull the current *devel* branch you'll have the correct modman file - sorry about that!

**Comment by dolito** [comment]
no problem, I used 0.6.2 ;)
just thought it would be good to fix this not functional release
